File: java-main/java-main/IO/Nestedfor5/p2.java
/*write a program to print the following pattern
# = = = =
= # = = =
= = # = =
= = = # =
= = = = #
*/

import java.util.Scanner;

class Demo {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        System.out.print("Enter the number of rows: ");
        int row = input.nextInt();

        for(int i = 1; i <= row; i++) {
            for(int j = 1; j <= row; j++) {
                if(i == j) {
                    System.out.print("# ");
                } else {
                    System.out.print("= ");
                }
            }
            System.out.println();
        }
    }
}

File: bootcamp_c_cpp_dsa-main/bootcamp_c_cpp_dsa-main/C/session5/p2.c
//            A
//         b  a
//      C  E  G
//   d  c  b  a
//E  G  I  K  M

#include<stdio.h>
void main()
{
	int row;
	printf("enter the rows:\n");
	scanf("%d",&row);
	char ch='A';
        for(int i=1;i<=row;i++){
		for(int sp=row;sp>i;sp--){
			printf(" \t");
		}
		char temp=ch;
		for(int j=1;j<=i;j++){
			if(i%2==1){
				printf("%c\t",temp);
				temp=temp+2;
			}else{
				printf("%c\t",temp+32);
				temp--;
			}
		}
		ch++;
		printf("\n");
	}
}
